qa/tasks/cephfs/test_admin.py

```diff
@@ -2345,5 +2345,5 @@ def test_with_health_warn_with_2_active_MDSs(self):
                               errmsgs=errmsg)
         self.negtest_ceph_cmd(args=f'mds fail {hw_mds_id}', retval=1,
                               errmsgs=errmsg)
-        self.run_ceph_cmd('mds fail mds1_id --yes-i-really-mean-it')
-        self.run_ceph_cmd('mds fail mds2_id --yes-i-really-mean-it')
+        self.run_ceph_cmd(f'mds fail {mds1_id} --yes-i-really-mean-it')
+        self.run_ceph_cmd(f'mds fail {mds2_id} --yes-i-really-mean-it')
```

Comment:
You already have these changes in your other PR?
https://github.com/ceph/ceph/pull/57492

Reply:
I've mentioned reason for it in description of PR #57492.